Tools/autotest/arducopter.py

```diff
@@ -2419,6 +2419,7 @@ def EK3_RNG_USE_HGT(self):
         self.reboot_sitl()
 
         # add a listener that verifies rangefinder innovations look good
+        global alt
```

Comment:
I wonder if "global" is a bit too wide a scope?

Reply:
Honestly don't know how this scoping works, and how to get a tighter one.  I do in Perl, but that's not going to help here ;-)

This is a pattern we use elsewhere, however.  So if we do work out how to fix it, we can do it in lots of places :-)

Reply:
OK!  fine with me.